src/layouts/Homepage/index.jsx

```diff
@@ -34,6 +34,7 @@ export default class Homepage extends DefaultTemplate {
       <Html>
         <Head
           title={this.props.file.title}
+          description={i18n.description}
```

Comment:
You should just use a direct meta & remove the specific code in the Head/index.js.

See https://github.com/putaindecode/putaindecode.fr/blob/735c24b574a0737b25d744f3a1d62b2a9e53e279/src/layouts/Post/index.jsx#L119-L130 for an example